api/base/exceptions.py

```diff
@@ -23,12 +24,14 @@ def json_api_exception_handler(exc, context):
                 else:
                     if isinstance(value, list):
                         for reason in value:
-                            errors.append({'detail': reason, 'meta': {'field': key}})
+                            errors.append({'source': {'pointer': '/data/attributes/' + key}, 'detail': reason})
                     else:
-                        errors.append({'detail': value, 'meta': {'field': key}})
+                        errors.append({'source': {'pointer': '/data/attributes/' + key}, 'detail': value})
```

Comment:
The only difference from the above branch seems to be the value of `'detail'`. Consider consolidating this to reduce redundancy.